cmd/commands/init.go

```diff
@@ -2,26 +2,38 @@ package commands
 
 import (
 	"io/ioutil"
+	"os"
 	"path"
 
-	"github.com/urfave/cli"
+	"github.com/spf13/cobra"
 
 	cmn "github.com/tendermint/go-common"
 )
 
-var InitCmd = cli.Command{
-	Name:      "init",
-	Usage:     "Initialize a basecoin blockchain",
-	ArgsUsage: "",
-	Action: func(c *cli.Context) error {
-		return cmdInit(c)
-	},
-	Flags: []cli.Flag{
-		ChainIDFlag,
-	},
+//commands
+var (
+	InitCmd = &cobra.Command{
+		Use:   "init",
+		Short: "Initialize a basecoin blockchain",
+		RunE:  initCmd,
+	}
+)
+
+// setupFile aborts on error... or should we return it??
+// returns 1 iff it set a file, otherwise 0 (so we can add them)
+func setupFile(path, data string, perm os.FileMode) (int, error) {
+	_, err := os.Stat(path)
+	if !os.IsNotExist(err) {
```

Comment:
causes errors when switch, this check is the same as in tendermint core, leaving (will add a comment)